generator/tsconfig.json

```diff
@@ -4,9 +4,10 @@
     "moduleResolution": "node",
     "checkJs": true,
     "allowJs": true,
+    "noEmit": true,
```

Comment:
Otherwise, I get an error about it can't output .js files because they're already authored in js.